src/common/dom/apply_themes_on_element.ts

```diff
@@ -21,13 +21,13 @@ const hexToRgb = (hex: string): string | null => {
  * element: Element to apply theme on.
  * themes: HASS Theme information
  * localTheme: selected theme.
- * updateMeta: boolean if we should update the theme-color meta element.
+ * mainElement: boolean if it is the top element of the page.
  */
 export const applyThemesOnElement = (
   element,
   themes,
   localTheme,
-  updateMeta = false
+  mainElement = false
 ) => {
   if (!element._themes) {
```

Comment:
We can add another quick bail out. If no `element._themes` defined and we use default theme.

Reply:
Shouldn't we always start with an empty object instead of using the old styles and override them? What if my new theme contains less than the old?